### PR TITLE
Added CSS to make Menu scrollable

### PR DESCRIPTION
**Pull request by rupin90**
Fix - https://www.awesomescreenshot.com/video/10559198?key=4ceabc29677dcf4bdae1a1a67b1c28ef

### CODE REVIEW

File: packages/web/components/templates/library/LibraryMenu.tsx [hunk 1]
```diff
@@ -2,13 +2,21 @@ import { VStack } from '../../elements/LayoutPrimitives'
...
 
   return (
-    <VStack alignment="start" distribution="start" css={{ width: '286px', pl: '16px', height: '100%' }}>
+    <VStack
+      alignment="start"
+      distribution="start"
+      css={{
+        width: '286px',
+        minWidth: '286px',
+        height: 'calc(100% - 100px)',
```
Comment (jacksonh) [inline packages/web/components/templates/library/LibraryMenu.tsx:15]: 🤯  nice trick, that is awesome.

**Review by jacksonh — APPROVED** [review]

**Review by jacksonh — APPROVED** [review]

**Review by jacksonh — APPROVED** [review]

**Comment by jacksonh** [comment]
Going to merge this one so I can see it on demo 🚀

File: packages/web/components/templates/library/LibraryMenu.tsx (hunk 1 above)
Comment (rupin90) [reply packages/web/components/templates/library/LibraryMenu.tsx:15]: It took a while haha 🙏🏽

**Comment by rupin90** [comment]
Cool, thank you!

**Pull request merged**